### PR TITLE
Adding Fields, Search Layouts and Customizations

### PR DESCRIPTION
**Pull request by dgamboni**
Internal Only Ticket to update the deploy_qa_config task and qa_org & regression_org flows in cci

# New Metadata

**EDA Settings**

- [ ] * EDA Settings > Affiliations > Enable "Specify Role for Created Affiliations" checkbox
- [ ] * EDA Settings > Affiliations > Enable "Copy Affiliation End Date from Program Enrollment:" and "Copy Affiliation Start Date from Program Enrollment:" checkboxes

**Attribute Object**

- Add “Subject Area” field to the Attribute Object, Credential Record Type Page Layout
- [ ] - Setup > Attribute > Page Layouts > EDA Credential Page Layout > Move “Subject Area” field underneath “Attribute Type” field
- Add Columns to the Attribute object “RECENTLY VIEWED” related list via Search Layouts
- [ ] Setup > Object Manager > Attribute > Search Layouts > Search Results > Edit > move the following from the left to the right and save
    * Attribute Name
    * Contact
    * Record Type
    * Attribute Type
    * Subject Area
    * Expired
    * End Date

Add Columns to the Attribute object “All” related list and pin it

- [ ] Go to “Attributes”, then click the wheel, click “Select Columns to Display”, choose these & save:
    * Attribute Name
    * Contact
    * Record Type
    * Attribute Type
    * Subject Area
    * Expired
    * End Date

**Contact Object**

Add ‘Chosen Full Name’ Field to HEDA Contact Page layout

- [ ] Setup > Object Manager > Contact > Page Layouts > HEDA Contact Layout > move “Chosen Full Name” under “Name” and save

Add ‘Attendance Event’ Related List to Contact Page layout

- [ ] Setup > Object Manager > Contact > Page Layouts > HEDA Contact Layout > Related Lists > move “Attendance Events” below “Addresses”

ADD ‘ATTRIBUTES’ RELATED LIST TO CONTACT PAGE LAYOUT

- [ ] Setup > Object Manager > Contact > Page Layouts > HEDA Contact Layout > Related Lists > move “Attributes” below “Attendance Events”

Add ‘Behavior Involvement’ Related List to Contact page Layout

- [ ] Setup > Object Manager > Contact > Page Layouts > HEDA Contact Layout > Related Lists > move “Behavior Involvement” below “Attributes”

Search Layouts

- [ ] Setup > Object Manager > Contact > Search Layouts > Search Results > Name, Account Name, Email, Primary Academic Program, Primary Language, Reports To

**Account Object**

Add the “_System: One2OneContact” field to the HEDA Household and HEDA Organizational Page Layouts

- [ ] Setup > Object Manager > Account > Page Layouts > move “_System: One2OneContact” field to both the Household and Organizational page layouts underneath “Account Record Type” on HH layout and “Phone” on Org Layout 

**Field Level Security**

 Give SysAdmin access to the display name on facility 

- [ ] Setup > Profiles > click “System Administrator” > FLS > Facility > Give read access

**Facility Object**

Add Display Name to the Facility Page Layout

- [ ] Obj Mgr > Facility > Page Layout > Display Name

Add Columns to Search Layout

**Term Object**

Configure Search Layout Columns

- [ ] Setup > Object Manager > Term > Search Results > Move these columns to the right and save :

    * Term Name, Parent Term, Type, Start Date, End Date

Term Grades Object

- [ ] Search Layouts

Course Offering Schedules

- [ ] Search Layouts - Course Offering Schedule Name, Course Offering, Time Block, Start Time, End Time

**Plan Requirements**

- [ ] Search Layouts

**Relationships**

- [ ] Search Layouts : Relationship Key, Contact, Related Contact, Reciprocal Relationship, Relationship Explanation, Description

**Course Object**

- [ ] Search layouts : Course Name, Course ID, Dept, Description

**Course Offering Object**

- [ ] Search Layouts

### CODE REVIEW

**Comment by robertbwatson** [comment]
Is this replacing #860?

**Comment by lifewithryan** [comment]
@dgamboni & @robertbwatson -- do we need/want all of those profiles? I realize its specific to QA, etc but since we don't distribute profiles or anything it seem superfluous.

**Comment by jstvz** [comment]
- @robertbwatson Closed #860. 
- @dgamboni Will add updates from [this comment](https://github.com/SalesforceFoundation/EDA/pull/860#issuecomment-510232587).
- @lifewithryan Nearly all of these profiles will be removed during cleanup.

**Comment by dgamboni** [comment]
Hi @jstvz I used the command you told me to get changes and also included all the changes in plain text here. This could replace #860.

File: unpackaged/config/qa/applications/___NAMESPACE___HEDA.app [hunk 1]
```diff
@@ -14,9 +14,4 @@
     <tabs>%%%NAMESPACE%%%Program_Plan__c</tabs>
     <tabs>%%%NAMESPACE%%%Trigger_Handler__c</tabs>
     <tabs>%%%NAMESPACE%%%HEDA_Settings</tabs>
-    <tabs>%%%NAMESPACE%%%Error__c</tabs>
```
Comment (robertbwatson) [inline unpackaged/config/qa/applications/___NAMESPACE___HEDA.app]: Are these intentionally being removed? I thought Dina originally added these to the app for easy access, just thought I'd ask here.

**Comment by jstvz** [comment]
@xinnanlisf Are you pulling these changes into #944?

**Comment by xinnanlisf** [comment]
Closing this one since this is handled in https://github.com/SalesforceFoundation/EDA/pull/944

**Pull request closed without merging**